package.json

```diff
@@ -61,20 +61,19 @@
     "prop-types": "^15.6.0",
     "query-string": "2.4.2",
     "raven-js": "3.19.1",
-    "react": "15.6.2",
+    "react": "16.2.0",
     "react-addons-css-transition-group": "15.6.2",
-    "react-addons-test-utils": "15.6.2",
-    "react-bootstrap": "^0.29.5",
-    "react-document-title": "1.0.4",
-    "react-dom": "15.6.2",
+    "react-bootstrap": "^0.32.0",
```

Comment:
nice, does this fix the warnings?

Reply:
i noticed it fixed some warnings relating to modals....